assets/runtime/env-defaults

```diff
@@ -6,6 +6,7 @@ DEBUG=${DEBUG:-$DEBUG_ENTRYPOINT}
 GITLAB_TEMP_DIR="${GITLAB_DATA_DIR}/tmp"
 GITLAB_BACKUP_DIR="${GITLAB_BACKUP_DIR:-$GITLAB_DATA_DIR/backups}"
```

Comment:
Is this variable not the same like you try to implement ?

Reply:
Backup dir is completly different.

See: https://docs.gitlab.com/ee/raketasks/backup_restore.html#specifying-a-custom-directory-for-backups

This new variable is for specifiying the subdiretory if you want to group you backups, also it is the only way to use a subdirectory within a bucket when using remote storage like S3.